worlds/cvcotm/items.py

```diff
@@ -0,0 +1,181 @@
+import logging
+
+from BaseClasses import Item
+from .data import iname
+from .locations import base_id
+
+from typing import TYPE_CHECKING, Dict, Union
+
+if TYPE_CHECKING:
+    from . import CVCotMWorld
+
+
+class CVCotMItem(Item):
+    game: str = "Castlevania Circle of the Moon"
+
+
+# # #    KEY    # # #
+# "code" = The unique part of the Item's AP code attribute, as well as the value to call the in-game "prepare item
+#          textbox" function with to give the Item in-game. Add this + base_id to get the actual AP code.
+# "default classification" = The AP Item Classification that gets assigned to instances of that Item in create_item
+#                            by default, unless I deliberately override it (as is the case for some Special1s).
+# "textbox id" = The ID of the textbox in-game that announces the item's receival.
+item_info = {
+    iname.heart_max:      {"code": 0xE400, "default classification": "filler"},
+    iname.hp_max:         {"code": 0xE401, "default classification": "filler"},
```

Comment:
You should probably add typing to this.

And if you don't mind a little refactoring, would recommend using a NamedTuple instead of dicts for the values here.

Quick example:
```
from typing import NamedTuple

class ItemData(NamedTuple):
    code: int
    default_classification: str

item_info: Dict[str, ItemData] = {
    iname.heart_max: ItemData(code=0xE400, item_classification="filler"),
    iname.hp_max: ItemData(0xE401, "filler"),
}
```